src/helpers/validationSchema.ts

```diff
@@ -3,14 +3,14 @@ import * as yup from "yup";
 export const validationSchema = {
   amount: yup
     .number()
-    .required("Skriv hvor meget du vil donere")
+    .required("Vælg hvor meget du vil donere")
     .min(1, "Mindst 1 kr.")
     .integer("Skriv et heltal")
     .typeError("Skriv et heltal"),
   recipient: yup.string().required("Vælg en modtager"),
   subscription: yup
     .string()
-    .required("Hvor ofte vil du donere?")
+    .required("Vælg en donationsfrekvens")
```

Comment:
@lindeloev does this sound okay in Danish? We weren't super sure 😅 The goal is to avoid duplication. We'll update in another PR if you have improvement ideas, for this or other texts 👍 

![image](https://user-images.githubusercontent.com/1177900/205174680-8b07b823-9f9f-40ce-8218-a8512b9b2b87.png)

Reply:
@maximbaz @mitnavn Good catch. But I think "donationsfrekvens" is a quite academic and still a duplicate semantically. Could we just say either [nothing] or "Vælg én"?